Makefile

```diff
@@ -24,6 +24,9 @@ all: docker
 gobuild:
 	./scripts/build false
 
+govet:
```

Comment:
can you move this down to line ~356 (https://github.com/aws/amazon-ecs-agent/pull/1810/files#diff-b67911656ef5d18c4ae36cb6741b7965R356) after gofmt and gocyclo? near where static-check related targets are?

Reply:
and add .PHONY

Reply:
sure.  adding now.